ci/cpu/prebuild.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+#Build rmm once per PYTHON
+export BUILD_RMM=1
+# if [[ "$CUDA" == "9.2" ]]; then
```

Comment:
Why did you comment these if statements out? They are required to prevent multiple uploads.

Reply:
> Why did you comment these if statements out? They are required to prevent multiple uploads.

Was for testing, woops! Will uncomment 😅